mina.core/core/src/test/java/org/apache/mina/transport/AbstractFileRegionTest.java

```diff
@@ -49,7 +49,10 @@
     protected abstract IoAcceptor createAcceptor();
     protected abstract IoConnector createConnector();
 
+    // Ignored for https://github.com/kaazing/gateway/issues/221
+    // Not sure how to ignore in junit.framework with inheritance
```

Comment:
Yeah, it looked like the test still ran.  Ignore was org.junit, but I couldn't find a junit.framework package equivilant